gufe/smallmoleculecomponent.py

```diff
@@ -205,3 +205,7 @@ def _from_sdf_supplier(cls, supp):
             raise RuntimeError(f"SDF contains more than 1 molecule")
 
         return cls(rdkit=mol)  # name is obtained automatically
+
+    @property
+    def formal_charge(self):
```

Comment:
As above, we only really care about the total charge of the component right?

Reply:
+1 -- I think total charge is the better term here.